intelmq/bots/parsers/eset/parser.py

```diff
@@ -60,7 +60,9 @@ def common_parse(self, event, line):
         event.add('classification.type', type)
         event.add('source.fqdn', line['domain'], raise_failure=False)  # IP addresses may be passed in line['domain']
 
-        event.add('source.ip', line['ip'])
+        event.add('source.ip', line['ip'], raise_failure=False)
```

Comment:
As we know that only `@ipv4_NXDOMAIN` and `@ipv6_NXDOMAIN` appear as values to be ignored, please specifically *only* ignore these. Otherwise we may actually miss something due to other errors.

Reply:
Done with `ip.endswith('_NXDOMAIN')`